spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'gets users' do
+      get '/'
+      expect(response).to have_http_status(200)
+      expect(response.body).to include('Here is list of users')
+    end
```

Comment:
- [ ] Great work testing for the HTTP status and the body. It will be great to ensure that the correct template is rendered also. So you should have a test that checks for this. For example: 

`expect(response).to render_template(:index)`